src/PKSim.Core/Mappers/ExpressionProfileToExpressionProfileBuildingBlockMapper.cs

```diff
@@ -0,0 +1,77 @@
+﻿using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Domain.Formulas;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Utility;
+using PKSim.Core.Model;
+using IFormulaFactory = PKSim.Core.Model.IFormulaFactory;
+
+namespace PKSim.Core.Mappers
+{
+   public interface IExpressionProfileToExpressionProfileBuildingBlockMapper : IMapper<ExpressionProfile, ExpressionProfileBuildingBlock>
+   {
+   }
+
+   public class ExpressionProfileToExpressionProfileBuildingBlockMapper : IExpressionProfileToExpressionProfileBuildingBlockMapper
+   {
+      private readonly IObjectBaseFactory _objectBaseFactory;
+      private readonly IEntityPathResolver _entityPathResolver;
+      private IFormulaFactory _formulaFactory;
+
+      public ExpressionProfileToExpressionProfileBuildingBlockMapper(IObjectBaseFactory objectBaseFactory, IEntityPathResolver entityPathResolver, IFormulaFactory formulaFactory)
+      {
+         _objectBaseFactory = objectBaseFactory;
+         _entityPathResolver = entityPathResolver;
+         _formulaFactory = formulaFactory;
+      }
+
+      public ExpressionProfileBuildingBlock MapFrom(ExpressionProfile expressionProfile)
+      {
+         var expressionProfileBuildingBlock = _objectBaseFactory.Create<ExpressionProfileBuildingBlock>();
+
+         expressionProfileBuildingBlock.Name = expressionProfile.Name;
+         expressionProfileBuildingBlock.PKSimVersion = ProjectVersions.Current;
```

Comment:
This  probably should be the Dipsplay version as opposed to the current which is a number that does not mean anything? I am not quite sure how this is supposed to be used but version 78 won't mean anything to anyone but us

Reply:
changed to ProjectVersions.Current.VersionDisplay. @msevestre that is the correct version right? like v11.1?

Reply:
@msevestre @rwmcintosh then actually we should also change PKSimVersion from int to string, right? Otherwise I would say the way this is written is how it was supposed to work....

Reply:
there also exists .VersionAsString

Reply:
ok, VersionDisplay is the correct one, changing to that